include/crypto/ecx.h

```diff
@@ -76,13 +76,12 @@ struct ecx_key_st {
 
 typedef struct ecx_key_st ECX_KEY;
 
-size_t ecx_key_length(ECX_KEY_TYPE type);
```

Comment:
Was this meant to be removed?

Reply:
yes, it looks like a leftover - it doesn't exist anywhere.